src/character.rs

```diff
@@ -30,3 +39,4 @@ impl_character!(u64);
 impl_character!(u32);
 impl_character!(u16);
 impl_character!(u8);
+impl_character!(usize);
```

Comment:
This is added for SA-IS, which recursively sorts the IDs (`usize`) of LSM-substrings found in the text.